### ISSUE TITLE
Add more crossover operators implementations for permutations 

### ISSUE THREAD

**Issue by ropinho** [nojhan/paradiseo#77]
Recently, I used the Paradiseo in a research project with many crossover operators for GAs using permutation encoding. Only OX and PMX exists in the Paradiseo, But others operators was implemented for the project, some generic operators and some operators that uses problem information.

I think would be interesting add the implementations of these generics crossover operators to the Paradiseo opening Pars with my code. The operators are:

* Alternating Edges Crossover (AEX) [Grefenstette et al. 1985]
* Cycle Crossover (CX) [Oliver et al. 1987]
* Linear Order Crossover (LOX) [Falkenauer and Bouffouix 1991]
* Modified Order Crossover (MOX) [Gwiazda et al. 2006]
* Precedence Preservative Crossover (PPX) [Mattfeld and Bierwirth 2004]

**Comment by nojhan** [comment]
I am not sure to understand your point, do you propose to add implementations that you made to Paradiseo, or do you say that you would need those operators?

**Comment by ronalddpinho** [comment]
My proposal is agregate implementations which I've made and used. 
If is interesting for the project, I can open a PR.

**Comment by nojhan** [comment]
Yes, I think it's definitely interesting! Please open a PR, and I will happily review (and merge) it.
